### PR TITLE
[ENH] Suffix type II: Grouping suffix 

### PR DESCRIPTION
**Pull request by agahkarakuzu**
* Add grouping suffix subsection
     * VFAT1 
     * IRT1 
     * MP2RAGE 
     * MEGRE
     * MESE 
     * MTR 
     * MTS 
     * MPM 
     * B1DAM

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -136,6 +136,45 @@ modalities include:
...
+|--------------------------------------------|---------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Variable flip angle                        | VFA     | Grouping | Parametrically linked anatomical images (primarily) for relaxometry mapping. The VFA method involves at least two spoiled gradient echo (SPGR) of steady-state free precession (SSFP) images acquired at different flip angles. Depending on the provided metadata fields and the sequence type, data may be eligible for DESPOT1, DESPOT2 and their variants. Please visit the [qMRI appendix](#prioritylevels) for details. _Associated output suffixes_: T1map, T2map, R1map, R2map |
```
Comment (Gilles86) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:149]: Where cna we find this qMRI appendix?

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -136,6 +136,45 @@ modalities include:
...
+| Inversion recovery (for T1 mapping)        | IRT1    | Grouping | Parametrically linked anatomical images for T1 mapping. The IRT1 method involves multiple inversion recovery spin-echo images acquired at different inversion times. _Associated output suffixes_: T1map, R1map                                                                                                                                                                                                                                                                                                                    |
+| Magnetization prepared two gradient echoes | MP2RAGE | Grouping | Parametrically linked anatomical images (primarily) for T1 mapping. The MP2RAGE method is a special protocol that collects several images at different flip angles and inversion times to create a parametric T1map by combining the magnitude and phase images. _Associated output suffixes_: T1map, R1map, UNIT1                                                                                                                                                                                                                 |
```
Comment (Gilles86) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:151]: Maybe put the original Weiskopf-reference here? What do your think? (I guess we should do the same for VFA, MP2RAGE, etc, though)

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -136,6 +136,45 @@ modalities include:
...
+Please see the [entity table appendix]() for the REQUIRED and OPTIONAL entities
+for each `grouping suffix`. 
+
+Note that every image in a **grouped scan collection** has the same `_<suffix>` 
+as they are likely to be used together.
+Although the acquisitions will have many identical acquisition parameters, only one-to-one-mapping is allowed between a `.json`-sidecar file and an image in a 
+grouped scan collection. This means that parameter values that are identical 
+across a set of grouped scans will still have to be stored separately in each `.json`-sidecar file.
+
+Note: Individual images within a grouped scan collection may be used for any purpose.
```
Comment (Gilles86) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:176]: I would leave this out. I don't think BIDS is about specifying how you are allowed to _use_ data.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md (hunk 1 above)
Comment (agahkarakuzu) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:149]: I'll push it after we clean up these PRs. You can find the appendix in the email I sent (zip file).

**Pull request merged**